scripts/init_chain_commands.sh

```diff
@@ -64,8 +62,12 @@ wait_count_blocks 2
 echo; echo "#### Voting on plans del proposal ####"
```

Comment:
im not sure you wanted to commit these changes.

Reply:
Same as last comment

Reply:
please dont commit to github local environment changes. :)